docs/platforms/perlmutter.rst

```diff
@@ -0,0 +1,180 @@
+
+Begin by loading the ``python`` module. The following modules are recommended::
+
+    module load PrgEnv-gnu cudatoolkit python
+
+Create a conda environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can create a conda_ environment in which to install libEnsemble and
+all dependencies. For example::
+
+    conda create -n libe-pm python=3.9 -y
+
+As perlmutter has a shared HOME filesystem with other clusters, using
+the ``-pm`` suffix (for perlmutter) is good practice.
+
+Installing libEnsemble and dependencies
+---------------------------------------
+
+Having loaded the Anaconda Python module, libEnsemble can be installed
+by one of the following ways.
+
+1. Install via **pip** into the environment.
+
+.. code-block:: console
+
+    (my_env) user@cori07:~$ pip install libensemble
+
+2. Install via **conda**:
+
+.. code-block:: console
+
+    (my_env) user@cori07:~$ conda config --add channels conda-forge
+    (my_env) user@cori07:~$ conda install -c conda-forge libensemble
+
+See :doc:`here<../advanced_installation>` for more information on advanced options
+for installing libEnsemble, including using Spack.
+
+Installing mpi4py
+^^^^^^^^^^^^^^^^^
+
+If using ``mpi4py`` for communications (optional), it is recommended that you install
+using the following line (having installed the ``cudatoolkit`` module)::
+
+    MPICC="cc -target-accel=nvidia80 -shared" pip install --force --no-cache-dir --no-binary=mpi4py mpi4py
+
+This line will build ``mpi4py`` on top of a CUDA-aware Cray MPICH.
+
+More information on using Python and ``mpi4py`` on Perlmutter can be found
+in the `Python on Perlmutter`_ documentation.
+
+Job Submission
+--------------
+
+Perlmutter uses Slurm_ for job submission and management. The two commands you'll
+likely use the most to initiate jobs are ``salloc`` and ``sbatch`` for running
+interactively and batch, respectively. libEnsemble runs on the compute nodes
+on Perlmutter using either ``multi-processing`` or ``mpi4py``.
+
+If running more than one worker per node the following is recommended to prevent
+resource conflicts::
+
+    export SLURM_EXACT=1
+    export SLURM_MEM_PER_NODE=0
+
+Alternatively, the ``--exact`` `option to srun`_, along with other relevant options can be given on
+any ``srun`` lines (including executor submission lines via the ``extra_args`` option).
+
+Example
+-------
+
+A simple example batch script for a libEnsemble use case that runs 5 workers (one
+generator and 4 simulators) on one node:
+
+.. code-block:: bash
+    :linenos:
+
+    #SBATCH -J libE_small_test
+    #SBATCH -A <myproject_g>
+    #SBATCH -C gpu
+    #SBATCH --time 15
+    #SBATCH --nodes 1
+
+    export MPICH_GPU_SUPPORT_ENABLED=1
+    export SLURM_EXACT=1
+    export SLURM_MEM_PER_NODE=0
+
+    python libe_calling_script.py --comms local --nworkers 5
+
+.. note::
+    Any modules loaded and environment variables (including conda environments) are
+    inherited by the job on Perlmutter. This is assumed in this example.
```

Comment:
What modules or variables are assumed when saying "This is assumed in this example"?

Reply:
I was thinking of the conda environment as I did not load in the script. In the above instructions it would be `libe_pm`. But I'll try to word more clearly.

Reply:
I was thinking of the conda environment as I did not load in the script. In the above instructions it would be `libe_pm`. But I'll try to word more clearly.

Reply:
Or just remove the last sentence.